analyses/copy_number_consensus_call/scripts/remove_dup_NULL_overlap_entries.py

```diff
@@ -52,6 +52,8 @@ def remove_dup_null_outside(cell, start, end):
 )
 parser.add_argument('--file', required=True,
                     help='path to the file that needs duplicates and NULLs removed')
+parser.add_argument('--uncalled', required=False,
```

Comment:
Is this used anywhere?

Reply:
No it is not. I had thoughts that I might use it to annotate where there were only two possible callers, but abandoned that plan on some reflection. I'll remove it unless there is some great idea for how to use the information in this step.